migrations/Version202608041023364106_pciSamples.php

```
@@ -0,0 +1,38 @@
+{
+    public function getDescription(): string
+    {
+        return 'Update TextReader IMS PCI to version 1.3.5: render ruby in page/nav labels, convert placeholders and sanitize button labels';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->runAction(new RegisterPciTextReaderIMS(), ['1.3.5']);
+    }
+
+    public function down(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)